.github/tests/angular-frontend-1/assertions.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -euxo pipefail
```

Comment:
Maybe we'll want to remove the `set -x` from those files (or at least from run-test.sh ?)

Reply:
Without the `-x` it wouldn't print which line the script failed.